target/linux/ipq806x/base-files/etc/board.d/02_network

```diff
@@ -52,6 +52,10 @@ linksys,ea8500)
 	ucidef_set_interface_macaddr "lan" "$hw_mac_addr"
 	ucidef_set_interface_macaddr "wan" "$hw_mac_addr"
 	;;
+mikrotik,rb3011uias)
```

Comment:
Please look at the other targets. We typically have names with "mikrotik,routerboard-something" ....